Bottom: fd0012f2d2114978488547b6e7c14d49c1ebb804
Top:    3b409221551a281a042ecfa60a6de3b76b1b69ec
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 12:53:53 +0100

hid/gtk: Turn on defines to debug a potential GTK-3.0 build


---

diff --git a/configure.ac b/configure.ac
index 71ab1c4..1150f22 100644
--- a/configure.ac
+++ b/configure.ac
@@ -770,6 +770,9 @@ $GTKGLEXT_PKG_ERRORS])]
 	GTKGLEXT_VER=`$PKG_CONFIG gtkglext-1.0 --modversion`
 	fi
 
+	GTK_CFLAGS="$GTK_CFLAGS -DGSEAL_ENABLE"
+	GTK_CFLAGS="$GTK_CFLAGS -DGDK_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED"
+
 	;;
 
       gcode|nelma|png )
